recipes/pangolin/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pangolin-opengl" %}
```

Comment:
Renamed from `pangolin-opengl` to avoid name conflict with other channels, suggested by https://github.com/conda-forge/staged-recipes/pull/26883#discussion_r1672848163

This feedstock will eventually have three outputs: C++ and Python bindings, as well as a meta package. The C++ binding will be named pangolin-cpp, the Python binding will be named pypangolin, and the meta package will be named pangolin-opengl, which depends on the other two. Please let me know if there is a better naming practice.

Reply:
Is it the same package that exists in bioconda? Or just a name clash?

Reply:
Just a name clash. The [package in bioconda](https://prefix.dev/channels/bioconda/packages/pangolin) refers to a different library ([link](https://github.com/cov-lineages/pangolin)) that has nothing to do with the package of this PR.